modules/ipi-install-verifying-support-for-redfish-apis.adoc

```diff
@@ -63,15 +63,25 @@ $ curl -u $USER:$PASS -X PATCH -H "Content-Type: application/json"  https://$Ser
 +
 [source,terminal]
 ----
-$ curl -u $USER:$PASS -X PATCH -H "Content-Type: application/json" https://$Server/redfish/v1/Systems/$SystemID/ -d '{"Boot": {"BootSourceOverrideTarget": "cd", "BootSourceOverrideEnabled": "Once"}}'
+$ curl -u $USER:$PASS -X PATCH -H "Content-Type: application/json" -H "If-Match: <ETAG>" https://$Server/redfish/v1/Systems/$SystemID/ -d '{"Boot": {"BootSourceOverrideTarget": "cd", "BootSourceOverrideEnabled": "Once"}}'
 ----
 
-. Check the ability to mount virtual media by running the following command:
+. Virtual media might use `POST` or `PATCH`, depending on your hardware. Check the ability to mount virtual media by running one of the following commands:
 +
 [source,terminal]
 ----
-$ curl -u $USER:$PASS -X PATCH -H "Content-Type: application/json" -H "If-Match: *" https://$Server/redfish/v1/Managers/$ManagerID/VirtualMedia/$VmediaId -d '{"Image": "https://example.com/test.iso", "TransferProtocolType": "HTTPS", "UserName": "", "Password":""}'
+$ curl -u $USER:$PASS -X POST -H "Content-Type: application/json" -H "If-Match: *" https://$Server/redfish/v1/Managers/$ManagerID/VirtualMedia/$VmediaId -d '{"Image": "https://example.com/test.iso", "TransferProtocolType": "HTTPS", "UserName": "", "Password":""}'
 ----
++
+[source,terminal]
+----
+$ curl -u $USER:$PASS -X PATCH -H "Content-Type: application/json" -H "If-Match: <ETAG>" https://$Server/redfish/v1/Managers/$ManagerID/VirtualMedia/$VmediaId -d '{"Image": "https://example.com/test.iso", "TransferProtocolType": "HTTPS", "UserName": "", "Password":""}'
+----
++
+[NOTE]
+====
+In some newer hardware, you might only find the `VirtualMedia` resource under `Systems/$SystemID` instead of `Managers/$ManagerID`.
```

Comment:
```suggestion
In some hardware, you might find the `VirtualMedia` resource under `Systems/$SystemID` instead of `Managers/$ManagerID`.
```
IBM Style warns against using "new": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#new. In this case, I think we can probably omit it. If it's important to indicate that we're specifically referring to newer hardware here, feel free to ignore my suggestion to remove "newer".

Reply:
I would agree with removing newer, since for example and R650 (which is from 2017) will have the VirtualMedia changed after doing firmware upgrades.